permissions/plugin-portshift-scanner.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "portshift-scanner"
+github: "jenkinsci/portshift-ci-plugin"
+paths:
+- "org/jenkins-ci/plugins/portshift-scanner"
+developers:
+- "portshift"
+security:
```

Comment:
`Please note that we generally reject email contacts due to the additional overhead in reaching out via email. Unless you represent a large organization with dedicated security team that needs to be involved in the coordination of a release, please refrain from requesting to be contacted via email.`